tensorboard/webapp/widgets/data_table/data_table_component.ng.html

```diff
@@ -17,6 +17,14 @@
     <div *ngIf="isContextMenuEmpty()" class="no-actions-message">
       No Actions Available
     </div>
+    <button
+      *ngIf="contextMenuHeader?.hidable"
```

Comment:
I don't think we should ever show both "Hide" and "Remove"

```suggestion
      *ngIf="!contextMenuHeader?.removable && contextMenuHeader?.hidable"
```